src/frontend/src/binder/backfill_order_strategy.rs

```diff
@@ -0,0 +1,99 @@
+// Copyright 2025 RisingWave Labs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::collections::HashMap;
+
+use risingwave_common::catalog::ObjectId;
+use risingwave_common::session_config::USER_NAME_WILD_CARD;
+use risingwave_pb::common::Uint32Vector;
+use risingwave_pb::stream_plan::backfill_order_strategy::Strategy as PbStrategy;
+use risingwave_pb::stream_plan::{
+    BackfillOrderFixed, BackfillOrderStrategy as PbBackfillOrderStrategy, BackfillOrderUnspecified,
+};
+use risingwave_sqlparser::ast::{BackfillOrderStrategy, ObjectName};
+
+use crate::Binder;
+use crate::catalog::CatalogError;
+use crate::catalog::schema_catalog::SchemaCatalog;
+use crate::error::Result;
+use crate::session::SessionImpl;
+
```

Comment:
Add some docs: why we don't use Binder, because binder stores relation context, and will lead to conflicts. It also binds a lot of extra stuff like system catalog etc..